packages/charts/src/components/ColumnChart/ColumnChart.tsx

```diff
@@ -211,10 +216,12 @@ const ColumnChart: FC<ColumnChartProps> = forwardRef((props: ColumnChartProps, r
                 axisLine={index < 1}
                 height={xAxisHeights[index]}
                 allowDuplicatedCategory={index === 0}
+                reversed={useIsRTL()}
               />
             );
           })}
         <YAxis
+          orientation={useIsRTL() === true ? 'right' : 'left'}
```

Comment:
please don't call hooks inside props. extract it to `const isRTL = useIsRTL()` somewhere before the return and just use the value here.